db/re/skill_db.yml

```diff
@@ -38997,29 +38997,29 @@ Body:
       - Level: 9
         Time: 500
       - Level: 10
-        Time: 300
+        Time: 700 # TODO: unknown cooldown for others levels
```

Comment:
I think it is not good that we keep 500 for level 9, when we have 700 for level 10.
Maybe it is better to do it like you did it for ABC_FRENZY_SHOT?